linkis-computation-governance/linkis-manager/linkis-application-manager/src/main/scala/org/apache/linkis/manager/am/service/em/DefaultEMEngineService.scala

```diff
@@ -125,6 +129,28 @@ class DefaultEMEngineService extends EMEngineService with Logging {
       instanceAndLabels.asScala.filter(
         _._1.getServiceInstance.equals(emInstanceLabel.getServiceInstance)
       )
+    } else if (clusterLabelOption.isDefined) { // follow the style of emInstanceLabel
+      val clusterLabel = clusterLabelOption.get.asInstanceOf[ClusterLabel]
```

Comment:
Cluster label does not need to make additional judgments, because it is the same as tenantLabel and other labels. The Feature.CORE type will already be the same instance of clusterLabel in the selected ecm.
org.apache.linkis.manager.label.service.impl.DefaultNodeLabelService#getScoredNodeMapsByLabels  
There is a strong consistent judgment on the label of the core